### PR TITLE
Migrated database to cloud

### LINKED ISSUE #19: Migrate database to cloud (MongoDB Atlas)

### PR DESCRIPTION
**Pull request by DebugAgrawal**
Resolves #19 
Added Mongo DB Atlas as database for SocioMark

### CODE REVIEW

**Comment by aitikgupta** [comment]
Can we do something around having a prod vs a dev environment, such that if we're using a dev env, then use the localhost db, otherwise cloud db?
(@bodhisha thoughts? Or this is fine?)

File: backend/app/server/database.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import motor.motor_asyncio
 
-MONGO_DETAILS = "mongodb://localhost:27017"
+MONGO_DETAILS = "mongodb+srv://sociomark:sociomark@userdb.7mwfe.mongodb.net/users?retryWrites=true&w=majority"
```
Comment (aitikgupta) [inline backend/app/server/database.py]: In any case, we should use the `.env` file to store this URL (even if it's localhost)

**Review by aitikgupta — CHANGES_REQUESTED** [review]
Until we decide on the above comment^

**Comment by aitikgupta** [comment]
Also, can you rebase to `main` branch?

**Comment by bodhisha** [comment]
> Can we do something around having a prod vs a dev environment, such that if we're using a dev env, then use the localhost db, otherwise cloud db?
> (@bodhisha thoughts? Or this is fine?)

Yup..this is possible and a good approach I guess. But not a priority

**Review by aitikgupta — APPROVED** [review]
👍🏼

**Review by bodhisha — APPROVED** [review]
Looks Good 🎉

**Pull request merged**